RubeesEat.UnitTests/Model/PersonTest.cs

```diff
@@ -0,0 +1,34 @@
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+[TestFixture]
+public class PersonTest
+{
+    [Test]
+    public void Person_Constructor_AssignsPropertiesCorrectly()
```

Comment:
As discussed, please remove the `Person_` prefix that was a mistake by me from the older comment. We just want to use `Constructor` as name for the first part of the test. So we had `Name_Setup_Expectation` (e.g. `Constructor_WithInvalidParameter_ThrowsException`). In some cases we can remove `Setup` and `Expectation` if there is a clear cut standard method execution. In the case of the constructor, this standard case is that the constructor executed without exceptions we can use just `Constructor` as name. If we had multiple constructors we could use `Setup` to differentiate (e.g. `Constructor_WithoutParameters` and `Constructor_WithName`).

Please go through all the tests and fix the naming accordingly.